luminoth/tools/server/web.py

```diff
@@ -1,26 +1,23 @@
-import os
-import json
 import click
 import tensorflow as tf
 
 from flask import Flask, jsonify, request, render_template
+from threading import Thread
 from PIL import Image
 
 from luminoth.utils.config import get_config
-from luminoth.utils.predicting import get_prediction
+from luminoth.utils.predicting import PredictorNetwork
 
 
 app = Flask(__name__)
 
-LOADED_MODELS = {}
-
 
 def get_image():
     image = request.files.get('image')
     if not image:
         return None
 
-    image = Image.open(image.stream)
+    image = Image.open(image.stream).convert('RGB')
```

Comment:
Shouldn't the `tf.errors.OutOfRangeError` exception be captured as above?

Reply:
The tf.errors.OutOfRangeError seems to be caused by opening an incompatible file with tf.gfile.Open, which is not an issue on the webserver.